plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/internal/crc/server/ui/CRCLocationSection.java

```diff
@@ -32,15 +32,15 @@
 
 public class CRCLocationSection extends AbstractLocationSection {
 
-	private static String SECTION_TITLE = "CRC Details";
-	private static String LABEL_STRING = "crc binary file: ";
-	private static String COMMAND_NAME = "Modify crc binary Location";
-	private static String LOC_ATTR = CRC100Server.PROPERTY_BINARY_FILE;
+	private static final String SECTION_TITLE = "CRC Details";
+	private static final String LABEL_STRING = "crc binary file: ";
```

Comment:
+1, but to be consistent across the board it would be even better to use all capital letters (or turn all occurrences to capital letter in the first place, all others in small letters)